src/run.ts

```diff
@@ -12,7 +12,7 @@ function runCellTests(app: JupyterFrontEnd, docManager: IDocumentManager): void
         buttons: [Dialog.cancelButton(), Dialog.okButton({ label: "Ok" })],
         title: "Run tests?",
     }).then((result) => {
-        if (result.button.label === "CANCEL") {
+        if (result.button.label === "Cancel") {
```

Comment:
This could also just be `!result.button.accept`, to avoid relying on the label text value (probably not API stable).